htcanalyze/main.py

```diff
@@ -562,20 +568,25 @@ def print_results(htcanalyze: HTCAnalyze,
         else:
             results = htcanalyze.analyzed_summary(log_files)  # else
 
-    # Allow this to happen
+    # Allow to end program without output, if no scripts were analyzed
     if results is None:
         sys.exit(0)
 
-    work_with = results
     # convert result to list, if given as dict
     if isinstance(results, dict):
         work_with = [results]
+    else:
+        work_with = results
 
     # check for ignore values
+    # TODO: I think, "mystery" should be named "result"?
+    # In any case, this name should describe at least something!
     for mystery in work_with:
 
         for i in mystery:
             if mystery[i] is None:
+                # TODO: also, if mystery is a word, that would appear in a log,
+                # this would not help a person looking into this log.
```

Comment:
double-space the comment, i.e., `#  this would ...`, as in `#__this would` where the _ is a space. That way, you ensure that if there's automatic coloring for TODO comments, it continues on the next line as well, as this is the line continuation for a comment.